src/components/rich-text-editor/rich-text-editor.ts

```diff
@@ -169,6 +215,14 @@ export class MRichTextEditor extends ModulVue implements InputManagementData, In
         }
     }
 
+    @Watch('firstHeaderLevel')
```

Comment:
au lieu d'utiliser 2 watch , tu peux te faire un compute (getter) qui retourne un boolean avec la condition. this.firstHeaderLevel > this.lastHeaderLevel.

Le compute va être mit à jour lorsqu'une des propriété a changé